app/assets/stylesheets/application.scss

```diff
@@ -24,17 +24,42 @@ body {
   background-size: cover;
 }
 
+footer {
+  height: auto;
+  background-color: black;
+}
+
+.banner-image {
+  padding-top: 15px;
+}
+
+.section-title {
+  font-family: 'Miltonian Tattoo', cursive;
+}
+
+.section-box {
+  background-color: gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.